apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -1,7 +1,7 @@
 import { gql, useMutation } from '@apollo/client';
```

Comment:
There's a mutation in this file, can you move it to where all the other mutations are stored?